crossdc-producer/src/test/java/org/apache/solr/crossdc/SolrAndKafkaMultiCollectionIntegrationTest.java

```diff
@@ -169,6 +181,14 @@ public void testFullCloudToCloudMultiCollection() throws Exception {
       solrCluster2.getSolrClient().request(create);
       solrCluster2.waitForActiveCollection(ALT_COLLECTION, 1, 1);
 
+      // Update the collection property "enabled" to true
+      CollectionProperties cp = new CollectionProperties(solrCluster1.getZkClient());
+      cp.setCollectionProperty(ALT_COLLECTION, "crossdc.enabled", "true");
+      cp.setCollectionProperty(ALT_COLLECTION, "crossdc.topicName", TOPIC);
```

Comment:
It seems like we're testing the fetching of the correct topic name, something that we can change at run time but I'm viewing this on the browser. Can you confirm?

Reply:
We just checking that properies besides enable (which is handled specially) also work by moving this from a system property to a collection property.